include/irr/asset/IAsset.h

```diff
@@ -6,155 +6,166 @@
 #include "irr/core/Types.h"
 #include "irr/core/refctd_dynamic_array.h"
 
-namespace irr { namespace asset
```

Comment:
what's happened here?

Reply:
it is probably because I copied and pasted entire code from my "backup file"